### PR TITLE
Naming Ex: Link to Figma and 1st UI

### PR DESCRIPTION
**Pull request by varya**

### CODE REVIEW

File: slides/ds101-worskhop.mdx [hunk 1]
```diff
@@ -137,6 +141,12 @@ Shared vocabulary is very important, it results in more productive cooperative w
 
 ---
 
+# Review
+
+[Mockups. The full version](https://www.figma.com/file/CCIfruTLVYWanYldiUvvYv/Naming-Ex?node-id=1%3A2)
```
Comment (varya) [inline slides/ds101-worskhop.mdx:194]: @okonet iframe is not allowed, so we cannot embed.

File: slides/ds101-worskhop.mdx [hunk 2]
```diff
@@ -119,6 +119,10 @@ There are mockups of small pieces of some interface. Each piece has two copies.
 
 Participants are divided into two groups: designers and developers. Designers and developers go to two different rooms where they get their tasks. Each participant gets a mockup of some interface. Mockups do not have any additional information, just a sketch of some UI. Participants get the following 5-minute tasks:
 
+Mockups:
+
+- [1](/assets/Naming-UI-1.png)
```
Comment (varya) [inline slides/ds101-worskhop.mdx]: There should be a way to host images and provide links to them. But these images should not be included into slides because participants get different ones.

Comment (okonet) [reply slides/ds101-worskhop.mdx]: I solved this in #13

File: slides/ds101-worskhop.mdx (hunk 1 above)
Comment (okonet) [reply slides/ds101-worskhop.mdx:194]: Solved in https://github.com/component-driven/ds-101-workshop/pull/24

File: slides/ds101-worskhop.mdx (hunk 2 above)
Comment (okonet) [reply slides/ds101-worskhop.mdx]: In #24 I introduced a way to use images directly from Figma so no export needed.

**Comment by okonet** [comment]
I think we can close this PR and continue in #24 or separate PRs. WYT?

**Comment by varya** [comment]
@okonet Please review again. Note that I force-pushed.

File: src/components/LayoutNaming.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import Layout from "./Layout";
+import NamingItem from "./NamingItem";
+
+const frame = "25%3A227";
+
+export default (props) => {
+  const images = useStaticQuery(graphql`
+    query NamingItemImageQuery {
+      allFigmaImage {
+        nodes {
```
Comment (okonet) [inline src/components/LayoutNaming.js:14]: Can we use the `eq` here instead of filtering below?

File: src/components/LayoutNaming.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { graphql, useStaticQuery } from "gatsby";
+import { Box, Flex, Grid } from "theme-ui";
+import { Link } from "gatsby";
+
+import { namingEx } from "../../config";
+import Layout from "./Layout";
+import NamingItem from "./NamingItem";
+
+const frame = "25%3A227";
```
Comment (okonet) [inline src/components/LayoutNaming.js]: Why is this needed?

File: src/components/LayoutNaming.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+          image
+        }
+      }
+    }
+  `);
+
+  const item = images.allFigmaImage.nodes.filter(
+    (i) => i.id === props.pageContext.frame
+  )[0];
+  console.log("LayoutNaming", props.pageContext.frame, item);
```
Comment (okonet) [inline src/components/LayoutNaming.js]: Remove?

File: src/pages/naming/index.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            context {
+              pageId
+            }
+          }
+        }
+      }
+    }
+  `);
+  let nodes = pages.allSitePage.edges;
+  console.log("pages", pages);
```
Comment (okonet) [inline src/pages/naming/index.js:24]: Remove?

File: src/components/LayoutNaming.js (hunk 3 above)
Comment (varya) [reply src/components/LayoutNaming.js:14]: Not sure. How to get access to pageContext data inside the query?

File: src/components/LayoutNaming.js (hunk 4 above)
Comment (varya) [reply src/components/LayoutNaming.js]: sorry, removed

File: src/pages/naming/index.js (hunk 6 above)
Comment (varya) [reply src/pages/naming/index.js:24]: removed

File: src/components/LayoutNaming.js (hunk 5 above)
Comment (varya) [reply src/components/LayoutNaming.js]: done

File: src/components/LayoutNaming.js (hunk 3 above)
Comment (okonet) [reply src/components/LayoutNaming.js:14]: Okay me neither so let’s keep and improve later

**Review by okonet — APPROVED** [review]
